tests/widget/operatorWithValidator/bondRewards/addBond.spec.ts

```diff
@@ -0,0 +1,185 @@
+        const nodeOperatorId = await widgetService.extractNodeOperatorId();
+
+        if (!nodeOperatorId) {
+          throw new Error('Node operator ID not found');
+        }
+
+        await test.step('Verify bond balance', async () => {
+          await expect(
+            bondRewardsPage.addBond.titledAmount.locator('div').first(),
+          ).toContainText('Bond balance');
+          const bondSummary =
+            await contractClients.CSAccounting.getBondSummary(nodeOperatorId);
+
+          await expect(bondRewardsPage.addBond.titledAmountBalance).toHaveText(
+            `${bondSummary.excess.toCut(4)} stETH`,
+          );
+        });
+
+        await test.step('Verify text', async () => {
+          const expectedText =
+            'Why you might need to add bond:Adding a bond serves as a voluntary security measure for your Node Operator to prevent your validators from becoming unbonded and being requested to exit in case of applied penalties.Supplied bond will be stored as stETH, which also garners staking rewards.';
+          await expect(bondRewardsPage.addBond.formInfoText).toContainText(
+            expectedText,
+          );
+
+          const expectedLinkHref =
+            'https://docs.lido.fi/staking-modules/csm/guides/unbonded-validators';
+          const linkUnbondedElement =
+            bondRewardsPage.addBond.formInfoText.locator('a');
+          // @todo: check to click on the link
+          await expect(linkUnbondedElement).toHaveAttribute(
+            'href',
+            expectedLinkHref,
+          );
+        });
+      });
+    },
+  );
+
+  [TOKENS.ETH, TOKENS.STETH, TOKENS.WSTETH].forEach((tokenName) => {
+    const tag = [Tags.performTX];
+    if (tokenName === 'STETH') tag.push(Tags.smoke);
+
+    test(
+      qase(193, `Should add bond using ${tokenName} as bond token`),
+      { tag },
+      async ({ widgetService, contractClients }) => {
+        qase.parameters({ tokenName });
+        const bondRewardsPage = widgetService.bondRewardsPage;
+
+        const nodeOperatorId = await widgetService.extractNodeOperatorId();
+
+        const expectedAmount = '0.0003';
+        const bondSummary =
+          await contractClients.CSAccounting.getBondSummary(nodeOperatorId);
+
+        await widgetService.addBond(tokenName, expectedAmount);
+
+        await test.step('Verify new balance after bond added', async () => {
+          const expectedBalance =
+            parseFloat(bondSummary.excess) + parseFloat(expectedAmount);
+          const actualBalance =
+            await bondRewardsPage.addBond.titledAmountBalance.textContent();
+
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+          expect(parseFloat(actualBalance!)).toBeCloseTo(expectedBalance);
+        });
+      },
+    );
+  });
+
+  [TOKENS.ETH, TOKENS.STETH, TOKENS.WSTETH].forEach((tokenName) => {
+    test(
+      qase(65, `Add bond using maximum available ${tokenName} amount`),
```

Comment:
Looks like test should be names "Check max button.."

Reply:
fixed